### ISSUE TITLE
Update lodash to version 4.17.13

### ISSUE THREAD

**Issue by DanielAglbts** [agilebits/emergency-kit#8]
A vulnerability has been found in versions < 4.17.13 of the dependency:

high severity
Vulnerable versions: < 4.17.13
Patched version: 4.17.13
Affected versions of lodash are vulnerable to Prototype Pollution.
The function defaultsDeep could be tricked into adding or modifying properties of Object.prototype using a constructor payload.

**Comment by DanielAglbts** [comment]
This is a submodule of `babel-cli`. Even updating that to the latest version would not update `lodash` to version 4.17.13: the latest version used by later versions of the package is 4.17.11 in [v7.4.3 (2019-04-02)](https://github.com/babel/babel/blob/master/CHANGELOG.md#v743-2019-04-02) of `babel-cli`.

**Comment by DanielAglbts** [comment]
This was fixed in the `babel` repo [today](https://github.com/babel/babel/commit/42000b99076577bae0f7ebbb803b04014f75da9a), but it requires us to update to v7 of the package.

**Comment by DanielAglbts** [comment]
I have updated all dependencies and fixed every security issue reported by `npm audit` in branch `daniel/dependencies`. However, there are two issues:

1. The latest version of `babel` and its subpackages have not been released to `npm` yet, so even after updating all dependencies (including `babel` to version 7, the latest version of `lodash` used in `package-lock.json` is only 4.17.11. I guess we will have to wait a bit longer for this.

2. While I was able to generate an Emergency Kit with the updated dependencies by running `node client-example.js`, building b5 fails with this error:

```
ERROR in /Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/emergency-kit/lib/emergency-kit.min.js
Module build failed: Error: unsupported type for static module: Property
at expression:

  fs: 9

    at traverse (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/static-module/index.js:301:34)
    at walk (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/static-module/index.js:224:18)
    at walk (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:49:9)
    at /Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:46:17
    at forEach (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/foreach/index.js:12:16)
    at walk (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:34:9)
    at /Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:41:25
    at forEach (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/foreach/index.js:12:16)
    at /Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:39:17
    at forEach (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/foreach/index.js:12:16)
    at walk (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:34:9)
    at /Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:41:25
    at forEach (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/foreach/index.js:12:16)
    at /Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:39:17
    at forEach (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/foreach/index.js:12:16)
    at walk (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:34:9)
    at /Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:46:17
    at forEach (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/foreach/index.js:12:16)
    at walk (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:34:9)
    at /Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:41:25
    at forEach (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/foreach/index.js:12:16)
    at /Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:39:17
    at forEach (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/foreach/index.js:12:16)
    at walk (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:34:9)
    at /Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:41:25
    at forEach (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/foreach/index.js:12:16)
    at /Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:39:17
    at forEach (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/foreach/index.js:12:16)
    at walk (/Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:34:9)
    at /Users/daniel/go/src/go.1password.io/b5/client/src/node_modules/falafel/index.js:46:17
 @ ./dist/esm/lib/emergency_kit.js 5:11-63
 @ ./dist/esm/locked/panels/team_join.react.js
 @ ./dist/esm/main/routes.js
 @ ./dist/esm/main/index.js
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! b5-react@1.0.0 wp: `node ../../node_modules/webpack/bin/webpack.js`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the b5-react@1.0.0 wp script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/daniel/.npm/_logs/2019-07-14T08_36_28_786Z-debug.log
wpApp failed. error: exit status 2
make: *** [all] Error 1
```

So it looks like some of the updated dependencies broke something. Therefore I am temporarily assigning this to @robyoder so he can take a look at why the code is failing.

**Comment by robyoder** [comment]
Resolved in #12
